vulcanai/net.py

```diff
@@ -409,6 +409,13 @@ def mse_loss(self, prediction, y):
         print("Using Mean Squared error loss")
         return lasagne.objectives.squared_error(prediction, y).mean()
 
+    def sensitive_cost(self, prediction, y):
+        print("Using Sensitive Cost Function")
+        sum_fn = np.sum(np.logical_and(prediction[:, 0] == 0, y[:,0] == 1))
```

Comment:
Yeah I was only trying to penalize the FN, since those are errors that we want to make sure we reduce (classifying someone who has suicidal ideations as not having them). If we penalize the FP class too, wouldn't that just be the same thing that the cross entropy function does?

Reply:
cross entropy simply looks at the difference in probability predicted vs actual. it won't take into account FN and FP since those are more complex abstract concepts. you would need to outline it yourself to get that sort of control over the training criterion.

Reply:
I'll add FP into the mix and see how the model performs then.